_posts/2021-01-29-status-update.md

```diff
@@ -0,0 +1,100 @@
+layout: post
+title: "2021-01-29: IAC Team Status Update"
+author: tphoney
+categories:
+  - team
+  - status
+tags:
+  - litmus
+  - cloud-ci
+  - modules
+  - iac-team
+  - community-day
+---
+
+Hey everyone, it's my honour to write my first blogpost for Puppet. Kinda strange in [I][TP] have done a number of other things for puppet, but never a blogpost. Case in point, my talk about [litmus](https://www.youtube.com/watch?v=FYfR7ZEGHoE) (available permanently on youtube to make me shudder at my own voice). On to business !!!
+
+## What is the IAC Team
+
+How did you find this post then ? Just kidding, we are a team at puppet that looks after all the modules / tasks / gems / tools that you as a community use. If we dont look after it we know who you should talk to. Here is a great blog post on how to reach us [Puppet IAC Team](https://puppetlabs.github.io/iac/team/2021/01/20/reaching-out.html)
+
+## Ecosystem Kickoff
+
+The IAC team is part of a larger group at puppet that has as its main focus 'you' the community. We had our opening meeting discussing the direction of our work for the next year, and it all looks very exciting. I can't give away too much yet, but more details will trickle out as we confirm our goals.
+
+## Automated Module Release
+
+What I can talk a little about is [Danny's][Danny] work on a new proof of concept. We are an automation company so we are always looking to remove a few steps from our process, this project will enable us to automate some of the manual steps required to release modules. Leaving the the human to only review and click merge on Github.
+
+## DSC
+
+Everyone's favourite topic, [Paula][Paula] and [Michael][Michael] have hit their first milestone, and are now focusing on the most important part. Documentation and blogposts ! There will also be a rebuild of the DSC modules to fix a vendored path issue. AND the march to 1.0 is well underway with a real focus on testing.
+
+## Networking Device Support Enhancements
+
+[Ciaran][Ciaran] has been working on enhancing the [cisco_ios](https://github.com/puppetlabs/cisco_ios) module to include two new [Bolt tasks](https://puppet.com/docs/bolt/latest/writing_tasks.html) to backup and restore configurations. The PoC has been proven out and now we're just wrapping up on some final development tasks before moving on to more extensive testing. We are looking to make this as universal as we can across as many Cisco devices as we can and have some exciting developments in the pipeline for better testing against Cisco devices. More details soon...
+
+## Community Contributions
+
+On the community day we normally focus on all the wonderful contributions we receive in Github, but over the past couple of weeks we have started to look at our JIRA backlog. There are a lot of tickets there as you can imagine. We have been categorising them based on the module they are associated with and also closing off tickets that are no longer relevant or that have already been fixed.
+
```

Comment:
May be we can give the actual numbers of PR and tickets looked on community day.

IAC Team processed 19 community PRs processed (9 tooling, 11modules) also we have looked at 31 module issues.

Reply:
that information is from the slide sheet ?

Reply:
yeah thats right its from the google sheet.